app/controllers/invoices_controller.rb

```diff
@@ -27,12 +30,31 @@ def create
    
     def show
         @invoice = Invoice.find(params[:id])
+        @client = @invoice
     end
     def edit
         @invoice = Invoice.find(params[:id])
     end
+    def update
+        @invoice = Invoice.find(params[:id])
+        #byebug
```

Comment:
Good practice to also remove all of these comments and debug artifacts when you clean up for a PR :)